src/libostree/libostree.sym

```diff
@@ -360,9 +360,14 @@ global:
  *                         NOTE NOTE NOTE
  */
 
+LIBOSTREE_2016.10 {
+global:
+        ostree_repo_regenerate_summary_ext;
+} LIBOSTREE_2016.8;
+
 /* Remove comment when first new symbol is added
-LIBOSTREE_2016.10
+LIBOSTREE_2016.11 {
```

Comment:
Maybe the patch is just weirdly rendered, but it looks like the new function gets called .10 and the old that .11.